### PR TITLE
Switch to src/

### LINKED ISSUE #50: Change Relative Routes For Alias ​​'src/' If Needed  /  Cambiar Las Rutas Relativas Para El Alias 'src/' En Caso De Ser Necesario
## Explanation:

Some components are importing stuff using some like this:

```js
import { element } from '../../../utils/logger'
```

I would like to change it to:

```js
import { element } from 'src/utils/logger'
```

## Context:

_Originally posted by @Israel-Laguan at https://github.com/Azordev/did-app/pull/49#discussion_r901022890_

---

# Spanish Version:

## Explicación:

Algunos componentes están importando cosas, utilizando algunos como este:

```js
importar { elemento } desde '../../../utils/logger'
```

Me gustaría  cambiarlo por

```js
importar { elemento } desde 'src/utils/logger'
```

## Contexto:

_Publicado originalmente por @Israel-Laguan en https://github.com/Azordev/did-app/pull/49#discussion_r901022890_

### PR DESCRIPTION
**Pull request by AdrianFagilde**
Change the " ../.../" by "src/"

### CODE REVIEW

**Review by Israel-Laguan — APPROVED** [review]

**Review by Eghinner — APPROVED** [review]

**Review by heliomar-pena — APPROVED** [review]
Good work!

File: src/actions/EventsActions.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { EventByPKReturnTypes } from 'src/utils/apollo.types';
+import { EventByPKReturnTypes } from './../utils/apollo.types';
```
Comment (heliomar-pena) [inline src/actions/EventsActions.ts]: Why not use src here?

File: src/actions/EventsActions.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 import { Notify } from 'quasar';
 import { EVENTS, EVENT_BY_ID_QUERY } from '../services';
```
Comment (heliomar-pena) [inline src/actions/EventsActions.ts]: Why not use src here?

File: src/actions/EventsActions.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
...
 import { useQuery, getListOfEventsReturnTypes, Event } from '../utils';
```
Comment (heliomar-pena) [inline src/actions/EventsActions.ts]: Why not use src here?

**Review by heliomar-pena — CHANGES_REQUESTED** [review]
Good work man! I just have some doubts

File: src/pages/Event/Event.layout.vue [hunk 4]
```diff
@@ -45,7 +45,7 @@
 
 <script lang="ts" setup>
 import { ref } from 'vue';
-import DetailLayout from '../../components/DetailLayout';
+import DetailLayout from 'src/components/DetailLayout';
 import { Event } from 'src/utils';
 import DIDLogo from '../../assets/logos/didperu-dark.svg';
```
Comment (heliomar-pena) [inline src/pages/Event/Event.layout.vue]:
```suggestion
import DIDLogo from 'src/assets/logos/didperu-dark.svg';
```

File: src/pages/Home/Home.layout.vue [hunk 5]
```diff
@@ -80,7 +80,6 @@ import { Event, EventType, Provider, User } from '../../utils';
 import { handleUserData } from 'src/utils';
 import DIDLogoDark from '../../assets/logos/didperu-dark.svg';
```
Comment (heliomar-pena) [inline src/pages/Home/Home.layout.vue]:
```suggestion
import DIDLogoDark from 'src/assets/logos/didperu-dark.svg';
```

**Review by heliomar-pena — APPROVED** [review]
Good work!

File: src/pages/Product/Product.layout.vue [hunk 6]
```diff
@@ -28,7 +28,8 @@
 </template>
 
 <script lang="ts" setup>
-import DetailLayout from 'src/components/DetailLayout';
+import DetailLayout from '../../components/DetailLayout';
```
Comment (heliomar-pena) [inline src/pages/Product/Product.layout.vue]:
```suggestion
import DetailLayout from 'src/components/DetailLayout';
```

File: src/pages/Home/index.vue [hunk 7]
```diff
@@ -4,9 +4,8 @@
 
 <script setup lang="ts">
 import HomeLayout from './Home.layout.vue';
-import { user } from './mock';
-import { getListOfEventsForHome, getListOfProviders } from 'src/actions';
-import { Event, Provider } from 'src/utils';
+import { getListOfEventsForHome, getListOfProviders } from '../../actions';
+import { Event, getUser, Provider, User } from '../../utils';
```
Comment (heliomar-pena) [inline src/pages/Home/index.vue]:
```suggestion
import { getListOfEventsForHome, getListOfProviders } from 'src/actions';
import { Event, getUser, Provider, User } from 'src/utils';
```

File: src/pages/Home/Home.layout.vue [hunk 8]
```diff
@@ -75,10 +75,11 @@
...
 <script setup lang="ts">
 import { HomeHeader, HomeList } from './components';
-import { ProviderCard, EventCard } from 'src/components';
-import { Event, EventType, Provider } from 'src/utils';
-import { handleUserData } from './utils/handleUserData';
-import DIDLogo from 'src/assets/logos/didperu-dark.svg';
+import { ProviderCard, EventCard } from '../../components';
+import { Event, EventType, Provider, User } from '../../utils';
+import { handleUserData } from 'src/utils';
+import DIDLogoDark from '../../assets/logos/didperu-dark.svg';
```
Comment (heliomar-pena) [inline src/pages/Home/Home.layout.vue]:
```suggestion
import { ProviderCard, EventCard } from 'src/components';
import { Event, EventType, Provider, User } from 'src/utils';
import { handleUserData } from 'src/utils';
import DIDLogoDark from 'src/assets/logos/didperu-dark.svg';
```

**Review by heliomar-pena — APPROVED** [review]
last changes

**Pull request merged**